packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py

```diff
@@ -294,6 +299,13 @@ def get_output_parameters_with_type(
             if metadata.repeated:
                 parameter_type = f"typing.Sequence[{parameter_type}]"
 
+        if metadata.message_type and metadata.message_type == "ni.protobuf.types.Double2DArray":
+            parameter_type = "Double2DArray"
+            custom_import_modules.append(_2DARRAY_DATA_IMPORT)
+
+            if metadata.repeated:
```

Comment:
We are not trying to support a repeated 2D array. I think this should throw an exception.

Reply:
Updated in the latest commit to raise exception if repeated is True